dspace/config/registries/eperson-types.xml

```diff
@@ -38,4 +38,17 @@
         <scope_note>Metadata field used for the language</scope_note>
     </dc-type>
 
+    <dc-type>
+        <schema>eperson</schema>
+        <element>orcid</element>
+        <scope_note>Metadata field used for the ORCID id</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>eperson</schema>
+        <element>orcid</element>
+        <qualifier>scope</qualifier>
+        <scope_note>Metadata field used for the granted ORCID scopes</scope_note>
+    </dc-type>
```

Comment:
Same question here. It seems like the EPerson object could just use `dspace.orcid.scope` metadata field (created in this PR) instead of creating yet another field.